### PR TITLE
Create an example mscs.defaults file if it is missing

### PR DESCRIPTION
**Pull request by sandain**

### CODE REVIEW

File: msctl [hunk 1]
```diff
@@ -1825,6 +1956,10 @@ elif [ -r "$HOME/mscs.defaults" ]; then
 elif [ -r "$HOME/.config/mscs/mscs.defaults" ]; then
   MSCS_DEFAULTS="$HOME/.config/mscs/mscs.defaults"
 fi
+if [ ! -r "$MSCS_DEFAULTS" ]; then
```
Comment (sandain) [inline msctl]: This should be `-s` (True if file exists and has a size greater than zero) rather than `-r` (True if file exists and is readable).

**Comment by sandain** [comment]
Any feedback here? This is a pretty simple change.

**Comment by zanix** [comment]
Looks alright to me.

**Comment by Roflicide** [comment]
Yes, this looks great!

**Pull request merged**